haystack/preview/components/rankers/meta_field.py

```diff
@@ -59,9 +59,9 @@ def __init__(
         if self.weight < 0 or self.weight > 1:
             raise ValueError(
                 """
-                Param <weight> needs to be in range [0,1] but was set to '{}'.\n
-                '0' disables sorting by metadata field, '0.5' gives equal weight to previous relevance scores and metadata field, and '1' ranks by metadata field only.\n
-                Please change param <weight> when initializing the MetaFieldRanker.
+                Parameter <weight> must be in range [0,1] but is currently set to '{}'.\n
+                '0' disables sorting by a metadata field, '0.5' assigns equal weight to the previous relevance scores and the metadata field, and '1' ranks by the metadata field only.\n
+                Change the <weight> parameter to a value in range o to 1 when initializing the MetaFieldRanker.
```

Comment:
This is supposed to be `0` right?
```suggestion
                Change the <weight> parameter to a value in range 0 to 1 when initializing the MetaFieldRanker.
```